### PR TITLE
PYIC-4086: add temp button to allow test for f2f delete details feature

### PR DESCRIPTION
**Pull request by shivanshuit914**
## Proposed changes

- Allow f2f delete jounery when featureflag `deleteDetailsEnabled` is enabled.

### What changed

- Added Temp button that we can use to test f2f delete details journey.

### Why did it change

- New feature development.

### Issue tracking

- [PYIC-4086](https://govukverify.atlassian.net/browse/PYIC-4086)

[PYIC-4086]: https://govukverify.atlassian.net/browse/PYIC-4086?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -441,7 +441,7 @@
         "insetText": "After you’ve deleted your details, your Post Office customer letter will no longer be valid and you should not try to use it to prove your identity.",
         "paragraph3": "We will automatically delete your details if you do not go to the Post Office within 10 days of getting your confirmation email.",
         "submitButtonText": "Continue to delete your details",
-        "paragraph4LinkText": "<a class=\"govuk-link\" href=\"/ipv/journey/next\" rel=\"noopener noreferrer\">I want to prove my identity at a Post Office</a>"
+        "paragraph4LinkText": "<a class=\"govuk-link\" href=\"/ipv/journey/end\" rel=\"noopener noreferrer\">I want to prove my identity at a Post Office</a>"
```
Comment (blakeyp) [inline src/locales/en/translation.json:444]: What's the reason for the change to `end` here?

Comment (shivanshuit914) [reply src/locales/en/translation.json:444]: Next redirects to `pyi-confirm-delete-details` page instead of `page-ipv-pending`  I think because of core-back changes.



```
  events:
    next:
      targetState:
        PYI_F2F_CONFIRM_DELETE_DETAILS_PAGE
    end:
      targetState:
        IPV_IDENTITY_PENDING_PAGE_TEST
```

Comment (blakeyp) [reply src/locales/en/translation.json:444]: Makes sense thanks

**Review by blakeyp — APPROVED** [review]
